packages/image_picker/image_picker_ios/example/ios/RunnerTests/ImagePickerPluginTests.m

```diff
@@ -170,13 +172,49 @@ - (void)testPickMultiImageShouldUseUIImagePickerControllerOnPreiOS14 {
 
   [plugin pickMultiImageWithMaxSize:[FLTMaxSize makeWithWidth:@(100) height:@(200)]
                             quality:@(50)
+                       fullMetadata:@(YES)
                          completion:^(NSArray<NSString *> *_Nullable result,
                                       FlutterError *_Nullable error){
                          }];
   OCMVerify(times(1),
             [mockUIImagePicker setSourceType:UIImagePickerControllerSourceTypePhotoLibrary]);
 }
 
+- (void)testPickImageWithoutFullMetadata API_AVAILABLE(ios(11)) {
+  id mockUIImagePicker = OCMClassMock([UIImagePickerController class]);
+  id photoLibrary = OCMClassMock([PHPhotoLibrary class]);
+
+  FLTImagePickerPlugin *plugin = [FLTImagePickerPlugin new];
```

Comment:
nits: do not use +new
https://google.github.io/styleguide/objcguide.html#do-not-use-new